### ISSUE TITLE
Definition - IFTA -> SON-EMU

### ISSUE THREAD

**Issue by felipevicens** [sonata-nfv/son-tests#241]
Create an integration test between IFTA and SON-EMU in order to authenticate and send a instantiation request from IFTA to SON-EMU

**Comment by mpeuster** [comment]
**Status update:**

The intention behind the test is to validate the integration between service platform and SDK emulator. The idea is to control the emulator and its `n` emulated PoPs (`n >= 1`)  with the SP. This will be done trough the emulator's fake OpenStack APIs that have already been used for the OSM integration.

The description of the test can be delivered in time. The actual implementation might be delayed. One reason for this is that the SP-to-emu integration is still ongoing work which is planned to be finalized end of April. See: https://github.com/sonata-nfv/son-tests/issues/241

**Comment by felipevicens** [comment]
@mpeuster You are right!. This task is only for definition. We can delay the implementation waiting for SP-to-emu integration.
